ahmed-yara-sara/src/components/Search.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from "react";
+import { Spinner, Form, FormControl, Button } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
You can import this line of code only in the app.js and it will  work for the whole app